draft-ietf-quic-transport.md

```diff
@@ -4361,42 +4361,43 @@ connection.
 
 # Transport Parameter Encoding {#transport-parameter-encoding}
 
-The format of the transport parameters is the TransportParameters struct from
-{{figure-transport-parameters}}.  This is described using the presentation
-language from Section 3 of {{!TLS13=RFC8446}}.
+The `extension_data` field of the quic_transport_parameters extension defined in
+{{QUIC-TLS}} contains the QUIC transport parameters. They are encoded as a
+sequence of transport parameters, as shown in {{transport-parameter-sequence}}:
```

Comment:
We could consider making this an ordered list. This would simplify the check that the peer didn't send any duplicate transport parameters.

Reply:
I'm in favor of that idea (especially because it reduces fingerprinting surface), but I'd like it to be a separate issue/PR for now.